packages/realm-network-transport/rollup.config.js

```diff
@@ -63,6 +63,21 @@ export default [
       nodeResolve(),
     ],
   },
+  {
+    input: "src/react-native/index.ts",
```

Comment:
We're adding a new "react-native" bundle to separate the code that runs on "react-native" from that which run on web.